ui/pages/create-account/connect-hardware/index.js

```diff
@@ -30,6 +30,7 @@ import ZENDESK_URLS from '../../../helpers/constants/zendesk-url';
 import { TextColor } from '../../../helpers/constants/design-system';
 import SelectHardware from './select-hardware';
 import AccountList from './account-list';
+import { getHardwareDeviceName } from '../../../store/actions';
```

Comment:
```suggestion
import { getHardwareDeviceName } from '../../../store/actions';
import SelectHardware from './select-hardware';
import AccountList from './account-list';
```
Lint
```
  33:1  error  `../../../store/actions` import should occur before import of `./select-hardware`  import/order
```
https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/102903/workflows/2ed4556b-be10-44b4-9c11-eb57dbe78150/jobs/3834349